pkg/aws/client.go

```diff
@@ -65,6 +65,8 @@ import (
 )
 
 // Name of the AWS user that will be used to create all the resources of the cluster:
+//
+//go:generate mockgen -source=client.go -package=aws -destination=mock_client.go
```

Comment:
@robpblake if i set package=mocks and destination=mocks/client.go i get errors about cycle imports in all the other pkg/aws files